app/src/main/java/com/github/se/cyrcle/ui/profile/EditProfileComponent.kt

```diff
@@ -52,14 +54,23 @@ fun EditProfileComponent(
   // To display the local profile picture when the user changes it, and hasn't uploaded (saved) it
   var hasTemporarilyChangedProfilePicture by remember { mutableStateOf(false) }
 
+  fun copyUser(user: User): User {
+    return user.copy(
+        public = user.public.copy(username = username),
+        details = user.details?.copy(firstName = firstName, lastName = lastName),
+        localSession =
+            user.localSession?.copy(profilePictureUri = profilePictureUri)
+                ?: LocalSession(profilePictureUri = profilePictureUri))
+  }
+
   val imagePickerLauncher =
       rememberLauncherForActivityResult(ActivityResultContracts.GetContent()) { uri ->
         uri?.let { profilePictureUri = it.toString() }
         hasTemporarilyChangedProfilePicture = true
       }
 
   Column(
-      modifier = Modifier.fillMaxSize().padding(16.dp).testTag("ProfileContent"),
+      modifier = modifier.fillMaxSize().padding(top = 15.dp).testTag("ProfileContent"),
```

Comment:
I don't think this test tag is used in the test. Is it intended ?

Reply:
Well spoted... But I don't think it can hurt to keep it, it might be usefull in later tests (E2E for example)

Reply:
I agree with you on this point , i was just checking that you didn't missed it during your tests even though you planned on using it